libs/datamodel/connectors/datamodel-connector/src/error.rs

```diff
@@ -30,6 +30,18 @@ pub enum ErrorKind {
     #[error("Native types are not supported with {} connector", connector_name)]
     ConnectorNotSupportedForNativeTypes { connector_name: String },
 
+    #[error(
+        "The prefix {} is invalid. It must equal the name of an existing datasource e.g. {}. Did you mean to use {}?",
```

Comment:
```suggestion
        "The prefix {} is invalid. It must equal to the name of an existing datasource e.g. {}. Did you mean to use {}?",
```

or 
```suggestion
        "The prefix {} is invalid. It must be equal to the name of an existing datasource e.g. {}. Did you mean to use {}?",
```

any one you prefer